src/styles/variables/_layout.scss

```diff
@@ -0,0 +1,80 @@
+//Layout variables
```

Comment:
Deze hele file staat in je variables folder, dat klinkt onlogisch. Je zou kunnen kijken naar ITCSS voor het goed structureren van je styling files die buiten je react componenten vallen.
https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/